### PR TITLE
sidebar: set active items one shade darker

### PR DESCRIPTION
**Pull request by rcrdlbl**
see: #924.

### CODE REVIEW

**Review by urcades — CHANGES_REQUESTED** [review]
This is very close, with exception to the behavior of hovering over an already-selected item/row in the sidebar — we should probably omit selected rows from the onHover styling treatment, as there's no additional utility gained from hovering over a selected row (in the sense that clicking a selected row won't actually do anything)

To get a rough sense of the feeling we're trying to evoke here, you can check out this figma, and hover/click over any of the **groups** in the sidebar (the other rows don't do anything).

**Comment by rcrdlbl** [comment]
@urcades updated!

**Review by urcades — CHANGES_REQUESTED** [review]
The edits I see made to the top nav items in both Groups and Talk are perfect, I only see one last change that needs to be made in the list of DMs/Groups themselves:

https://user-images.githubusercontent.com/1195363/194147824-511e1a4e-bf07-444c-8e7c-d7774cd57d74.mov

You can see in the first part of this  video, the on-hover and highlighting behavior is as-specified, but as I hover the individual DMs, they remain butted up against one another — their onHover styling is perfect though.

All that remains is ensuring the small spacing is inherited by these row items as well!

**Review by urcades — APPROVED** [review]
Ignore the previous comment, @rcrdlbl alerted me to the fact that this PR is simply about the active item shading, not the row spacing.

**Comment by rcrdlbl** [comment]
The latest commit should address your concern, @urcades

File: ui/tsconfig.rvpu8p5fi8d.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (arthyn) [inline ui/tsconfig.rvpu8p5fi8d.json:1]: Remove this file

**Review by arthyn — CHANGES_REQUESTED** [review]
Just the one file to remove, then good to merge

File: ui/tsconfig.rvpu8p5fi8d.json (hunk 1 above)
Comment (rcrdlbl) [reply ui/tsconfig.rvpu8p5fi8d.json]: Done!

**Review by arthyn — APPROVED** [review]
lgtm

**Pull request merged**